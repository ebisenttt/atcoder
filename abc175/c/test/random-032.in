-885155160348229 709205095159703 492629676512368
