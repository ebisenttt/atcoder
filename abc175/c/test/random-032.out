392525483835861
